packages/opc-footer/README.md

```diff
@@ -0,0 +1,99 @@
+# This is opc-footer 👋
+
+Web component based on lit-html
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+
+the opc-footer have supports dark theme, by default the light theme is selected. for example
+footer with default theme:
+
+```html
+<opc-footer></opc-footer>
+```
+
+![Demo Footer light no links](images/Footer light no links.png)
+
+here's another with dark
+
+```html
+<opc-footer theme="dark"></opc-footer>
+```
+
+![Demo Footer dark no links](images/Footer dark no links.png)
+
+To add the links it has a attribute called opc-links-groups. this supports the Array folloing is the structure.
+
+```js
+[
+  {
+    name: "Example Group Link",
+    links: [
+      {
+        name: "Crunch time",
+        path: "https://example.com",
+      },
+      {
+        name: "Lorem ipsum Link",
+      },
+    ],
+  },
+  ...
+];
+```
+For a demo example see the
+![Demo Footer light with links](demo/images/Footer-light-with-links.png)
+![Demo Footer dark with links](demo/images/Footer-dark-with-links.png)
+
+The Links with no path fires the custom event called footer-link-click, so the user can add
+custom behaviors.
+
+for example 
+```js
+  const myFooterElement = document.querySelector('opc-footer');
+  myFooterElement.addEventListener('opc-footer-link-click', (e) => {
+    console.log(e);
+    /// custom behaviors.
+  });
+```
+
+### Development server
+
+- Install packages
+
+```sh
+npm run install
+```
+
+- Run development server
+
+```sh
+npm run dev
+```
+
+OR
+
+```sh
+npm run start
+```
+
+### Build
+
+```sh
+npm run build
+```
+
+## Run tests
+
+```sh
+npm run test
+```
+
+## 🤝 Contributors
+
+👤 **Sumeet Ingole**
```

Comment:
Also mention the license, since you've already added the MIT license file.

Although, I'm not sure if license's are needed for each individual component if it is already present in the library parent directory.
@diwanshi @sayak-sarkar What do you think?